streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1258,7 +1259,7 @@ public void cancel(Application appParam) throws Exception {
                 savePoint.setAppId(application.getId());
                 savePoint.setLatest(true);
                 savePoint.setType(CheckPointType.SAVEPOINT.get());
-                savePoint.setTriggerTime(now);
```

Comment:
Could the `now` be removed?

Reply:
thx~ 
@1996fanrui did you mean that the `Date now = new Date()` should be inlined into `savePoint.setCreateTime(new Date())` ?

Reply:
No, is the `now` used after your PR?